engine/access/rest/transactions_test.go

```diff
@@ -62,13 +62,16 @@ func validCreateBody(tx flow.TransactionBody) map[string]interface{} {
 			"key_index":       fmt.Sprintf("%d", tx.ProposalKey.KeyIndex),
```

Comment:
Generally speaking this bug make me think we don't have enough tests on complex transactions with potentially bad data. It should be done as part of new PR in the future for sure.